src/bindings.ts

```diff
@@ -22,4 +22,10 @@ export interface Bindings {
 
 	// Worker version metadata
 	VERSION_METADATA: ScriptVersion;
+
+	// Key rotation schedule
+	ROTATION_DAY_OF_MONTH: string;
+	ROTATION_DAY_OF_WEEK: string;
+	ROTATION_HOUR_OF_DAY: string;
+	ROTATION_CRON_STRING?: string;
```

Comment:
I would say, let's stick to only one method (cron string only?)

Reply:
Just added support for cron strings, lmk if we should only leave this one or add a Date string based on this: https://github.com/cloudflare/privacypass-issuer/pull/43#discussion_r1688261775